lib/jorb/backend.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Jorb.Backend do
+  @moduledoc ~S"""
+  The backend behavior represents a queueing abstraction.
```

Comment:
Erlang/Elixir behaviours use the UK spelling.

Reply:
Just because I have to use the UK spelling in the syntax doesn't mean I have to use it in my comments

Reply:
![kenny powers america fuck you](https://user-images.githubusercontent.com/109716/36796816-7dd4ca36-1c74-11e8-83cf-983dfb989dfd.gif)